openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -10,12 +10,42 @@
 from .models import BlockStructureConfiguration
 
 # Switches
+# .. toggle_name: INVALIDATE_CACHE_ON_PUBLISH
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Invalidates block structure cache when changes to courses are published.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14571
 INVALIDATE_CACHE_ON_PUBLISH = WaffleSwitch(
     "block_structure.invalidate_cache_on_publish", __name__
 )
+
+# .. toggle_name: STORAGE_BACKING_FOR_CACHE
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Enables retrieving block structures from storage instead of regenerating
+#   block structures when they aren't available in cache.
+# .. toggle_warnings: Depends on the storage class, `STORAGE_CLASS`, and storage keyword arguments,
```

Comment:
If you use something like the following, they could potentially search the docs more easily for details.
```
Depends on BLOCK_STRUCTURES_SETTINGS['STORAGE_CLASS'] and ...
```

Reply:
Yeah, I don't know what I was thinking to be honest 😅 Addressed this 👍🏼